model_overview_simple_network_sim.md

```diff
@@ -0,0 +1,89 @@
+# Model overview:  simple_network_sim 
+**Basics**
+The model simple_network_sim is a network-based simulation model built for COVID-19 and written in Python. 
+
+The core team includes:
+* Jess Enright (modelling lead)
+* Paul Johnson (epidemiologist)
+* Bob Turner (research software engineer)
+
+Development is supported by contributing volunteer engineers and developers. 
+
+
+
+**Nodes-as-populations vs nodes-as-individuals**
+There are currently two versions of the model suitable at different scales under development.  
+
+In the ***nodes-as-populations*** version, each node contains an age-structured population and fractional people within that population are recorded as in various disease states.  We assume that the scale of the simulation is such that individuals do not progress between age classes.   As we will see in the next section, disease progression in this version is largely deterministic.  This version is currently considered the core version of this model, and is likely to be usable at larger population scales. 
+
+In the ***nodes-as-individuals*** version, each node is a single infectious unit or individual, and can only have one disease state.  This version is far more stochastic, is currently considered an alternate version, and is likely to be more appropriate at finer scale. 
+
+**Disease progression compartmental model**
+We use a compartmental model including the states:
+* S: susceptible
+* E: exposed, but not yet infectious
+* A: asymptomatic infectious individuals
+* I: symptomatic infectious individuals
+* H: hospitalised individuals 
+* R: recovered (assumed immune)
+* D: dead
+
+Movement between **S** and **E** is mediated by the infectious process model, all other transitions are determined by an input transition matrix that encodes the rates along the arrows in the following diagram:
+
+![](https://i.imgur.com/2rrJ6Qm.png)
+
+Different rates by age class are possible. 
```

Comment:
Is this the case for all transitions or just S->E? Forgive me if this is a dumb question.

Reply:
It's possible for all transitions.